README.rst

```diff
@@ -144,14 +137,14 @@ is produced, something is wrong.
 By default, tests are prevented from overusing resources like disk space and
 memory.  To enable these tests, run ``make testall``.
 
-IMPORTANT: If the tests fail and you decide to mail a bug report, *don't*
+IMPORTANT: If the tests fail and you decide to file a bug report, *don't*
 include the output of ``make test``.  It is useless.  Run the failing test
 manually, as follows::
```

Comment:
This paragraph is a bit harsh, from the uppercase "IMPORTANT" to the calling the output of "make test" "useless".

I would rewrite it as:

If the tests fail, you can run the failing test(s) in verbose mode using:
    ./python -m test -v test_that_failed
You should then file a bug report and include the output of this command.